deploy/charts/emqx-operator/templates/admission-webhooks/patch-job/certgen/job-patchWebhook.yaml

```diff
@@ -0,0 +1,63 @@
+{{- if not  (index .Values "admissionWebhooks" "cert" "cert-manager" "enable") -}}
+{{- if .Values.admissionWebhooks.cert.certgen.enable -}}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ include "emqx-operator.fullname" . }}-certgen-patch
+  namespace: {{ .Release.Namespace }}
+  annotations:
+    "helm.sh/hook": post-install,post-upgrade
+    "helm.sh/hook-delete-policy": before-hook-creation,hook-succeeded
```

Comment:
Looks some hooks has `hook-succeeded`, but some hooks didn't has `hook-succeeded`, what's the different ?